lib/ansible/template/__init__.py

```diff
@@ -744,7 +744,7 @@ def _lookup(self, name, *args, **kwargs):
                     ran = wrap_var(ran)
                 else:
                     try:
-                        ran = UnsafeProxy(",".join(ran))
+                        ran = AnsibleUnsafeText(u",".join(ran))
```

Comment:
i would limit the use of the classes and stress `wrap_var` as the general interface that deals with all 'unsafing'